portal/src/graphql/portal/AuthenticationConfigurationScreen.tsx

```diff
@@ -28,6 +28,16 @@ const authenticationScreenQuery = gql`
   }
 `;
 
+function isPortalApiApp(value: any): value is PortalAPIApp {
```

Comment:
I suppose this one can be replaced too?

Reply:
This one cast the value PortalAPIApp, the condition only let typescript infer the type `Record<string, unknown>` in this case, and the overlap is not enough for casting directly. So there is 2 options

1. typeguard
2. `as any as ...`

and I found 1 looks nicer @kiootic

Reply:
It seems the `Record<string, unknown>` you got comes from the `GQL_AppConfig` scalar type. We should update it in `graphql.d.ts` to use our expected app config type `PortalAPIAppConfig`, i.e.: `declare type GQL_AppConfig = PortalAPIAppConfig;`.

Reply:
it works, thanks